tests/libcxx/skipped_tests.txt

```diff
@@ -212,26 +229,36 @@ containers\unord\unord.set\max_size.pass.cpp
 
 # Extreme compiler memory consumption.
 utilities\tuple\tuple.tuple\tuple.apply\apply_large_arity.pass.cpp
+utilities\tuple\tuple.tuple\tuple.cnstr\recursion_depth.pass.cpp
 
 # .verify.cpp tests use clang-verify to validate warnings
+atomics\atomics.types.generic\trivially_copyable.verify.cpp
+atomics\atomics.types.operations\atomics.types.operations.req\copy.assign.ptr.volatile.verify.cpp
+atomics\atomics.types.operations\atomics.types.operations.req\copy.assign.volatile.verify.cpp
+concepts\concepts.lang\concept.default.init\default_initializable.verify.cpp
 containers\associative\map\map.access\empty.verify.cpp
 containers\associative\multimap\empty.verify.cpp
 containers\associative\multiset\empty.verify.cpp
 containers\associative\set\empty.verify.cpp
 containers\container.adaptors\priority.queue\priqueue.members\empty.verify.cpp
 containers\container.adaptors\queue\queue.defn\empty.verify.cpp
 containers\container.adaptors\stack\stack.defn\empty.verify.cpp
+containers\container.node\node_handle.nodiscard.verify.cpp
 containers\sequences\array\empty.verify.cpp
 containers\sequences\deque\deque.capacity\empty.verify.cpp
 containers\sequences\forwardlist\empty.verify.cpp
 containers\sequences\list\list.capacity\empty.verify.cpp
-containers\sequences\vector.bool\empty.verify.cpp
 containers\sequences\vector\vector.capacity\empty.verify.cpp
 containers\sequences\vector\vector.cons\copy.move_only.verify.cpp
+containers\sequences\vector.bool\empty.verify.cpp

```

Comment:
There is major divergence between `expected_results.txt` and `skipped_tests.txt` in this entire section. Some of it is definitely unnecessary (e.g. the order of `containers/sequences/vector.bool/empty.verify.cpp`). There are other differences I don't understand - e.g. numerous tests are listed only for `skipped_tests.txt`, but why do we need to skip them for internal Contest only? (I note that skipping too many tests will silently succeed.)

If it's intentional that we're skipping things for internal Contest only, I would expect them to appear in the `# *** SKIPPED FOR MSVC-INTERNAL CONTEST ONLY ***` section at the end (which avoids cluttering up a diff between `expected_results.txt` and `skipped_tests.txt`).




Note about ordering: I don't really care about the order in which tests are listed here - VSCode sorting is nice but not critical - however, I strongly believe that we should avoid unnecessary divergence between these files, so that we can easily maintain them in parallel.

Reply:
The GitHub runner knows about `*.verify.cpp` and `*.sh.cpp`, so both sections of skips should be in the Contest-only section of `skipped_tests.txt`.